src/pretix/static/pretixpresale/scss/_event.scss

```diff
@@ -187,18 +187,19 @@ article.item-with-variations .product-row:last-child:after {
 div.front-page {
     margin-top: 30px;
 }
-.offline-banner {
-    background: $brand-danger;
-    color: white;
+.offline-banner, .timemachine-banner {
```

Comment:
timemachine-banner no longer exists, does it?

Reply:
```suggestion
.offline-banner {
```